extensions/wikidata/src/org/openrefine/wikidata/qa/scrutinizers/MultiValueScrutinizer.java

```diff
@@ -3,15 +3,36 @@
 import org.openrefine.wikidata.qa.QAWarning;
 import org.openrefine.wikidata.updates.ItemUpdate;
 import org.wikidata.wdtk.datamodel.interfaces.PropertyIdValue;
+import org.wikidata.wdtk.datamodel.interfaces.Snak;
+import org.wikidata.wdtk.datamodel.interfaces.SnakGroup;
 import org.wikidata.wdtk.datamodel.interfaces.Statement;
+import org.wikidata.wdtk.datamodel.interfaces.Value;
 
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
 public class MultiValueScrutinizer extends EditScrutinizer {
 
     public static final String new_type = "multi-valued-property-is-required-for-new-item";
     public static final String existing_type = "multi-valued-property-is-required-for-existing-item";
+    public static String MULTI_VALUE_CONSTRAINT_QID = "Q21510857";
+    public static String MULTI_VALUE_CONSTRAINT_STATUS = "P2316";
+
+    class MultivalueConstraint {
+        Value constraintStatus;
+
+        MultivalueConstraint(PropertyIdValue propertyIdValue) {
```

Comment:
I would rather pass a `Statement` as parameter, and do the initial fetching externally. This ensures that an instance of this inner class corresponds to a constraint declaration.